src/bin/abstract-sql-compiler.ts

```diff
@@ -11,6 +11,7 @@ import { program } from 'commander';
 
 const runCompile = (inputFile: string, outputFile?: string) => {
 	const { generateSqlModel } =
+		// eslint-disable-next-line @typescript-eslint/no-var-requires
```

Comment:
I'm wondering if we want to replace the requires with await import in that case?
```suggestion
	const { generateSqlModel } = (await import(
		'../sbvr-api/sbvr-utils'
	)) as typeof SbvrUtils;
```

Reply:
@fisehara Doing that would require marking the function as an async one, which I guessed that @Page- would have already done that if possible. After trying it though it seems that it works fine when running the commends from the bC api scripts (unless I'm missing anything) so great idea :+1:

Reply:
Here is an extra PR for you to review and try :)
See: https://github.com/balena-io/pinejs/pull/683